OneSignalSDK/onesignal/src/main/java/com/onesignal/OSInAppMessage.java

```diff
@@ -54,6 +54,7 @@
     private OSInAppMessageDisplayStats displayStats = new OSInAppMessageDisplayStats();
 
     private double displayDuration;
+    private boolean displayed = false;
```

Comment:
`displayed` seems to generic and is confusing. Thinking it should be called `displayedInSession`, and updating all other usages in this PR to match.